api/health/service.go

```diff
@@ -33,12 +38,24 @@ func (s *Service) Readiness(_ *http.Request, _ *struct{}, reply *APIReply) error
 }
 
 // Health returns a summation of the health of the node
-func (s *Service) Health(_ *http.Request, _ *struct{}, reply *APIReply) error {
+func (s *Service) Health(_ *http.Request, args *HealthArgs, reply *APIReply) error {
 	s.log.Debug("API called",
 		zap.String("service", "health"),
 		zap.String("method", "health"),
 	)
-	reply.Checks, reply.Healthy = s.health.Health()
+
+	if args == nil || len(args.SubnetIDs) == 0 {
```

Comment:
I don't think this can be `nil` -- we don't have this check in any other service